generated/niswitch/niswitch/_grpc.py

```diff
@@ -0,0 +1,313 @@
+            grpc_types.GetAttributeViInt32Request(vi=self._vi, channel_name=channel_name, attribute_id=attribute_id),
+        )
+        return response.attribute_value
+
+    def get_attribute_vi_real64(self, channel_name, attribute_id):  # noqa: N802
+        response = self._invoke(
+            self._client.GetAttributeViReal64,
+            grpc_types.GetAttributeViReal64Request(vi=self._vi, channel_name=channel_name, attribute_id=attribute_id),
+        )
+        return response.attribute_value
+
+    def get_attribute_vi_string(self, channel_name, attribute_id):  # noqa: N802
+        response = self._invoke(
+            self._client.GetAttributeViString,
+            grpc_types.GetAttributeViStringRequest(vi=self._vi, channel_name=channel_name, attribute_id=attribute_id),
+        )
+        return response.attribute_value
+
+    def get_channel_name(self, index):  # noqa: N802
+        response = self._invoke(
+            self._client.GetChannelName,
+            grpc_types.GetChannelNameRequest(vi=self._vi, index=index),
+        )
+        return response.channel_name_buffer
+
+    def get_error(self):  # noqa: N802
+        response = self._invoke(
+            self._client.GetError,
+            grpc_types.GetErrorRequest(vi=self._vi),
+        )
+        return response.code, response.description
+
+    def get_path(self, channel1, channel2):  # noqa: N802
+        response = self._invoke(
+            self._client.GetPath,
+            grpc_types.GetPathRequest(vi=self._vi, channel1=channel1, channel2=channel2),
+        )
+        return response.path
+
+    def get_relay_count(self, relay_name):  # noqa: N802
+        response = self._invoke(
+            self._client.GetRelayCount,
+            grpc_types.GetRelayCountRequest(vi=self._vi, relay_name=relay_name),
+        )
+        return response.relay_count
+
+    def get_relay_name(self, index):  # noqa: N802
+        response = self._invoke(
+            self._client.GetRelayName,
+            grpc_types.GetRelayNameRequest(vi=self._vi, index=index),
+        )
+        return response.relay_name_buffer
+
+    def get_relay_position(self, relay_name):  # noqa: N802
+        response = self._invoke(
+            self._client.GetRelayPosition,
+            grpc_types.GetRelayPositionRequest(vi=self._vi, relay_name=relay_name),
+        )
+        return enums.RelayPosition(response.relay_position_raw)
+
+    def init_with_topology(self, resource_name, topology, simulate, reset_device):  # noqa: N802
+        response = self._invoke(
+            self._client.InitWithTopology,
+            grpc_types.InitWithTopologyRequest(resource_name=resource_name, topology=topology, simulate=simulate, reset_device=reset_device),
+        )
+        return response.vi
+
+    def initiate_scan(self):  # noqa: N802
+        self._invoke(
+            self._client.InitiateScan,
+            grpc_types.InitiateScanRequest(vi=self._vi),
+        )
+
+    def lock(self):  # noqa: N802
```

Comment:
Waiting on grpc-device regen of the proto file now that hapigen is updated with the lock/unlock metadata. Once that is done will update and regen everything here.

Reply:
This is now updated